src/ProjectFileTools/extension.vsixmanifest

```diff
@@ -13,6 +13,7 @@
   <Assets>
     <Asset Type="Microsoft.VisualStudio.VsPackage" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
     <Asset Type="Microsoft.VisualStudio.MefComponent" Path="|ProjectFileTools.NuGetSearch|" />
+    <Asset Type="Microsoft.VisualStudio.MefComponent" Path="|ProjectFileTools.MSBuild|" />
```

Comment:
We ended up going with a different workaround for that dll, and I believe we do not need it listed as an asset now. I cleared out all of the bin/obj files and the project builds and runs successfully (I also made the change above in the ProjectFileTools/source.extension.vsixmanifest  file).